Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "log4rs"
 version = "1.3.0"
-authors = ["Steven Fackler <sfackler@gmail.com>", "Evan Simmons <esims89@gmail.com>"]
+authors = [
```

Comment:
I think you need to revert changes in the formatting

Reply:
Changes have been approved by the maintainers, I'm reluctant to push to the PR at this stage for that reason